workshops/README.md

```diff
@@ -1,81 +1,19 @@
-
-Our workshops are free to be used by anyone. If you see any issues,
-[make an issue](https://github.com/hackedu/hackedu/issues/new) or even better
-yet, make a
-[pull request](https://help.github.com/articles/using-pull-requests/)!
-
-Unofficial workshops are in the [`contrib`](contrib/) directory.
-
---------------------------------------------------------------------------------
-
-## Portfolio
-
-![](img/portfolio.png)
-
-Learn to build a simple portfolio website (see example above).
-
-| Attribute | Value                                        |
-| ----------| -------------------------------------------- |
-| Audience  | Beginners who have never written HTML before |
-| Duration  | 45 mins - 1.5 hours                          |
-| Concepts  | Basics of HTML & CSS                         |
-
-[![](img/open.png)](portfolio/README.md)
-[![](img/facilitation_guidelines.png)](workshop_details.md#portfolio)
-
-## Remote Control
-
-![](img/remote_control.png)
-
-Learn to make a web app that calls your friend and plays a song when you press a
-button. You will also learn to easily send text messages and make phone calls
-from the browser.
-
-### Important Note
-
-You will need to obtain and set up a Twilio account before the workshop. Luckily
-we have a promotion code that will give you $100 worth of credit. Just follow
-the instructions in the
-[facilitation guidelines](workshop_details.md#remote-control) for details. Give
-yourself at least an hour to do this.
-
-There are two parts to this workshop. Part I is linear
-and guided while Part II is exploratory.
-
-| Attribute          | Value                                                                                                                                                   |
-| -------------------| ------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| Audience           | Beginners who have never written a line of code before and veteran programmers alike (veterans will love the ease of use of the powerful API). |
-| Duration of Part I | 30 mins - 1.5 hours                                                                                                                                     |
-| Duration of Part II | 30 mins - 3 hours                                                                                                                                       |
-| Concepts           | Send text messages, make phone calls, play music, detect when a button was clicked, detect when your cell phone moves                                   |
-
-[![](img/open.png)](remote_control/README.md)
-[![](img/facilitation_guidelines.png)](workshop_details.md#remote-control)
-
-## Frogger
-
-![](img/frogger_win.gif)
-
-Learn to build Frogger.
-
-[![](img/facilitation_guidelines.png)](workshop_details.md#frogger)
+<h1 id="personal-website">Personal Website</h1>
```

Comment:
@paked Whoops, was testing out something called classeur :( -- thanks for pointing that out :)